test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -290,7 +290,7 @@
                   (update-in [:headers] assoc "x-client-cert" "NOCERTS"))
           {:keys [status body]} (app req)]
       (is (= status 400))
-      (is (= body "No certs found in PEM read from x-client-cert")))))
+      (is (re-matches #".*No certs found in PEM read from x-client-cert.*" body)))))
```

Comment:
This doesn't look necessary.

Thinking back on it I believe I updated the [middleware here](https://github.com/puppetlabs/trapperkeeper-authorization/pull/38/files#diff-983d1c3afdb0656721ce0c5b3f4f8049R45) to use the default behavior of `wrap-bad-request` (output json), updated the test to check that the content was in there without concern for its format. Realized I shouldn't be changing the format if at all possible, updated the middleware to output plain text, but forgot to revert the test change.

I can revert this test change, or add an assertion that the content type is also in plain text.